module/zfs/metaslab.c

```diff
@@ -3556,8 +3658,18 @@ metaslab_group_alloc_normal(metaslab_group_t *mg, zio_alloc_list_t *zal,
 			kmem_free(search, sizeof (*search));
 			return (-1ULL);
 		}
-
 		mutex_enter(&msp->ms_lock);
+
+		metaslab_active_mask_verify(msp);
+
+		/*
+		 * This code is commented out because of issues with
+		 * tracepoints in non-gpl kernel modules.
+		DTRACE_PROBE3(ms__activation__attempt,
+		    metaslab_t *, msp, uint64_t, activation_weight,
+		    boolean_t, was_active);
+		 */
```

Comment:
I agree disabling them for now makes good sense, but you'll need to do it in a way which keeps the style checker happy.  Aside from that, looks good.